### DISCUSSION TITLE
Feat/product collection images

### DISCUSSION THREAD

**Discussion by edihasaj** [medusajs/medusa#3076] [Feature Requests]
Last year I added image support for product collections that I show in my store, do you guys think this is good solution?

Here is the PR: https://github.com/medusajs/medusa/pull/2483
PR for ADMIN: https://github.com/medusajs/admin/pull/748

Drived from this: https://github.com/medusajs/medusa/pull/1799

**Comment by tiltroom** [comment]
I think this is very nice. I have implemented something similar myself.

**Comment by edihasaj** [comment]
I am not sure if we are going to merge those features to the core or not, at least to have a response so that we could seek other solutions (that are not that good though)... what do you thing @olivermrbl ?

**Reply by tiltroom** [reply]
Well, this is open source so anyone could use this even if it's not implemented in the production branch. I guess this is the true flexibility of Medusa. I have seen your other PRs, all great ideas and things I needed and implemented my own way too 😄 , too bad I didn't see your contributions first, it would've saved me a lot of time.

**Reply by edihasaj** [reply]
Yeah, but why re-inventing the wheel.. I understand OSS I was just suggesting, since for every version we need to merge and stuff, while this can be base functionality. I don't know it's just my opinion of course 🤷
